pkgs/top-level/all-packages.nix

```
@@ -35971,6 +35971,7 @@ with pkgs;
   go-exploitdb = callPackage ../tools/security/go-exploitdb { };
 
   groestlcoin  = libsForQt5.callPackage ../applications/blockchains/groestlcoin {
+    stdenv = if stdenv.isDarwin then darwin.apple_sdk_11_0.stdenv else stdenv;
```

```suggestion
    stdenv = darwin.apple_sdk_11_0.stdenv;
```
for linux that variable is equal to stdenv.